spaceship/lib/spaceship/two_step_or_factor_client.rb

```diff
@@ -4,6 +4,7 @@
 module Spaceship
   class Client
     def handle_two_step_or_factor(response)
+      raise "2FA can only be performed in interactive mode" unless interactive
```

Comment:
```suggestion
      raise "2FA can only be performed in interactive mode" if ENV["FASTLANE_IS_INTERACTIVE"]
```

We like to keep `fastlane_core`  out of `spaceship` so this solution is pretty okay! But I'm curious if maybe we do something a little bit different and look for an environment variable called `FASTLANE_IS_INTERACTIVE`

And then maybe we do something like... 
```rb
# Set interactive environment variable for spaceship (which can't require fastlane_core)
ENV["FASTLANE_IS_INTERACTIVE"] = FastlaneCore::UI.interactive?
``

in https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/cli_tools_distributor.rb#L73

Thoughts? 😊

Reply:
Makes sense to me. As `ENV` can only hold `String` values I compare the value to `false` to allow `spaceship` execution in case it's run outside of Fastlane.